docker_images/fasttext/tests/test_api_text_classification.py

```diff
@@ -87,10 +91,11 @@ def test_malformed_question(self):
 
     def test_multiple_words(self):
         inputs = "this is great"
-
+        
+        expected_status_code = 200 if "language-identification" in self.model_id else 400
```

Comment:
Maybe add a docstring explaining why this is the case

Reply:
I added a comment why this is the case (since I didn't see docstring used elsewhere) but let me know if you meant docstring!